.gitignore

```diff
@@ -2,3 +2,5 @@ config.yml
 yet-another-cloudwatch-exporter
 vendor
 dist
+
+.idea/*
```

Comment:
Could you please remove the change in the .gitignore files. They are not project specific in my opinion.

You could use https://help.github.com/en/github/using-git/ignoring-files for that :)

Reply:
@thomaspeitz - removed - makes sense and thank you